app/features/onboarding/views/forms/_ci_bot_account.erb

```diff
@@ -19,8 +19,15 @@
 
   <ol>
     <li>
-      <a href="https://help.github.com/articles/signing-up-for-a-new-github-account/">
+      <a href="https://help.github.com/articles/signing-up-for-a-new-github-account/" target="_blank">
         Create an account</a> for your <code>fastlane.ci</code> bot on GitHub.
+
+        <p>		
+          <i>		
+            Note: Accounts with two-factor authentication are currently not supported.
```

Comment:
Looking at the other issues, we should support 2FA bots just fine. We should enforce the use of personal access tokens over the actual password, see discussion at https://github.com/fastlane/ci/issues/406

Reply:
This is a great change. Do you want to merge this and then alter the documentation when 406 is built and merger ?

Reply:
#406 doesn't have to be merged, I think we just have to be sure to always call it "Personal access token" in the UI, and never "password" and we should be fine for now 👍

Reply:
Also, let me know if you want to join the fastlane slack, just send me your email via DM or email 👍

Reply:
Will do :) 

By merged I mean “until there’s a PR to address these language changes” , feel free to close this if unneeded

Reply:
I kinda wanted to highlight the "As your password", since otherwise that terminology is super-confusing. It says "password" everywhere. To be honest I wouldn't mind doing a PR to change everything to from `FASTLANE_CI_PASSWORD` to `FASTLANE_CI_PERSONAL_ACCESS_TOKEN`

Reply:
Hah, I understand. That's why @AndrewMcBurney is working on some of that in https://github.com/fastlane/ci/pull/661

We're talking about 2 different things. fastlane.ci requires your bot to have a fastlane.ci account. That account needs a fastlane.ci password, and a github api token. fastlane.ci also requires you to have a fastlane.ci account for your own user, and that user must also have a fastlane.ci password, and a github api token. 

This is all because 1) we need a place to store your github token (inside your fastlane.ci user) and 2) because github doesn't yet offer an authentication mechanism for people who can't use `callback urls` for App Auth.

Reply:
@taquitos Didn't see that PR! Cool. I made a tiny change in ea536b9. Let me know if this makes more sense to you :)